File: Lista01/q1.java
public class q1{
    public static void main(String [] args){
        int i;
        double[] vetor =  {1,2,3,4,5};
        double medidos = 0, variacao = 0;
        for(i=0; i<5; i++){
            medidos += vetor[i];
        }

        medidos/=5;

        double []vetorDif = new double[5];

        for(i=0; i<5; i++){
            vetorDif[i] = (vetor[i] - medidos) * (vetor[i] - medidos);
            variacao += vetorDif[i];
        }

        variacao/=5;

        double result = Math.sqrt(variacao);

        System.out.println("Variacao: " + variacao + "\nMedia: " 
        + medidos + "\nDesvio: " + result);

        if (result > (medidos * 0.1)) {
            System.out.println("O multimetro esta com problemas");
        }else{
            System.out.print("O multimetro nao esta com problemas");
        }
}

}

File: Lista02/q1.py
import math

vetor = [1,2,3,4,5]
media = 0

for i in range(5):
    media+= vetor[i]

media/=5

vetDif = []
variacao = 0

for i in range(5):
    dif = vetor[i] - media
    vetDif.append(math.pow(dif,2))
    variacao += vetDif[i];

variacao/=5

result = math.sqrt(variacao)

print("Variacao: " + str(variacao) + "\nMedia: " + str(media) + "\nDesvio: " + str(result))
